app/src/androidTest/java/mozilla/lockbox/Navigator.kt

```diff
@@ -38,14 +42,46 @@ class Navigator {
         activityRule.activity.runOnUiThread {
             activityRule.activity.recreate()
         }
+    }
+
+    fun disconnectAccount() {
+        val subject = DataStore.shared.state as ReplaySubject<DataStore.State>
+        when (subject.value) {
+            is DataStore.State.Locked -> {
+                Dispatcher.shared.dispatch(DataStoreAction.Unlock)
+                blockUntil(DataStore.State.Unlocked)
+            }
+
+            is DataStore.State.Unprepared -> {
+                return
+            }
+
+            else -> {
+                // NOP
```

Comment:
Should this be empty?

Reply:
Yes. The `// NOP` is short for noop. We should probably make this comment more like '// do nothing`, minimally.